firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/Datagrams.cpp

```diff
@@ -183,7 +183,7 @@ SCENARIO(
     // as payload buffer is a ByteVector of size 252
     int payload_size = GENERATE(0, 252);
 
-    for (size_t i = 0; i < payload_size; ++i) {
+    for (int i = 0; i < payload_size; ++i) {
```

Comment:
Any particular reason you changed the counter variable `i` in these `for` loops from `size_t` to `int`?

Reply:
so I had declared `payload_size` as an int for some reason, that's a catch2 generator and the compiler complained about `comparison of integer expressions of different signedness`. Now I'm using both as `size_t` instead, thus reverting the change `i` as an int